extensionsGallery-insider.json

```diff
@@ -5162,14 +5162,15 @@
 					},
 					"versions": [
 						{
-							"version": "1.0.0",
-							"lastUpdated": "08/23/2023",
+					
+							"version": "0.0.3",
+							"lastUpdated": "09/04/2024",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.SQLOps.DownloadPage",
-									"source": "https://github.com/lozn00/AzureSQLProcConvertSQL/raw/master/StoredProcedureConverter-0.0.1.vsix"
+									"source": "https://github.com/lozn00/AzureSQLProcConvertSQL/releases/download/v0.0.3/StoredProcedureConverter-0.0.3.vsix"
```

Comment:
Please update the other content URLs as well - and ideally pin them to the tag version. 

e.g.

`https://raw.githubusercontent.com/lozn00/AzureSQLProcConvertSQL/v0.0.3/LICENSE`

(note that the original filename in this case was wrong so that needs to be fixed. Make sure you check all the other links too)